subprojects/language-scala/src/integTest/groovy/org/gradle/language/scala/CachedPlatformScalaCompileIntegrationTest.groovy

```diff
@@ -25,6 +25,10 @@ class CachedPlatformScalaCompileIntegrationTest extends AbstractCachedCompileInt
     String compilationTask = ':compileMainJarMainScala'
     String compiledFile = "build/classes/main/jar/Person.class"
 
+    def setup() {
+        executer.withFullDeprecationStackTraceDisabled()
```

Comment:
Why is this necessary?

Reply:
It's not necessary, but it's close to undebuggable when you have dozens of stacktraces to wade through.